neuralmonkey/vocabulary.py

```diff
@@ -126,7 +128,10 @@ def from_dataset(datasets: List[Dataset], series_ids: List[str], max_size: int,
                 vocabulary.add_tokenized_text(
                     [token for sent in series for token in sent])
 
-    vocabulary.trunkate(max_size)
+    vocabulary.truncate(max_size)
+
+    if min_freq > 1:
```

Comment:
This will fail if `min_freq` is `None`. Either remove `Optional` from the type hint or add `is not None` to the condition.